views/lexicon/patterns/tab-header/table.html.twig

```diff
@@ -0,0 +1,304 @@
+                'Owner': 'Paul Verhoven',
+                'Modified': '2012-08-29 12:00',
+                'Live': true,
+                'Visible': true
+            },
+            {
+                'Title': 'Maggie',
+                'Owner': 'Henry Hobson',
+                'Modified': '2015-05-06 12:00',
+                'Live': true,
+                'Visible': true
+            },
+            {
+                'Title': 'Red Sonja',
+                'Owner': 'Richard Fleischer',
+                'Modified': '1985-07-03 12:00',
+                'Live': true,
+                'Visible': true
+            },
+            {
+                'Title': 'Kindergarten Cop',
+                'Owner': 'Ivan Reitman',
+                'Modified': '1991-02-01 12:00',
+                'Live': true,
+                'Visible': true
+            },
+            {
+                'Title': 'The Running Man',
+                'Owner': 'Paul Michael Glaser',
+                'Modified': '1987-11-13 12:00',
+                'Live': true,
+                'Visible': true
+            },
+            {
+                'Title': 'Junior',
+                'Owner': 'Ivan Reitman',
+                'Modified': '1994-11-23 12:00',
+                'Live': true,
+                'Visible': true
+            },
+            {
+                'Title': 'Last Action Hero',
+                'Owner': 'John McTiernan',
+                'Modified': '1993-07-30 12:00',
+                'Live': true,
+                'Visible': true
+            },
+            {
+                'Title': 'Red Heat',
+                'Owner': 'Walter Hill',
+                'Modified': '1998-06-17 12:00',
+                'Live': true,
+                'Visible': true
+            },
+            {
+                'Title': 'Raw Deal',
+                'Owner': 'John Irvin',
+                'Modified': '1986-06-06 12:00',
+                'Live': true,
+                'Visible': true
+            },
+            {
+                'Title': 'Eraser',
+                'Owner': 'Chuck Russell',
+                'Modified': '1996-06-21 12:00',
+                'Live': true,
+                'Visible': true
+            },
+            {
+                'Title': 'Jingle All the Way',
+                'Owner': 'Brian Levant',
+                'Modified': '1996-11-16 12:00',
+                'Live': true,
+                'Visible': true
+            },
+            {
+                'Title': 'The 6th Day',
+                'Owner': 'Roger Spottiswoode',
+                'Modified': '2000-11-13 12:00',
+                'Live': true,
+                'Visible': true
+            },
+            {
+                'Title': 'Collateral Damage',
+                'Owner': 'Andrew Davis',
+                'Modified': '2002-02-08 12:00',
+                'Live': true,
+                'Visible': true
+            }
+        ]
+    }
+%}
+
+{% block tab_header %}
+    <p>tab header goes here</p>
```

Comment:
Could we add this to the settings panel too, with it having it's own settings specific flex styling. Would be helpful for future layout testing.